types/address_test.go

```diff
@@ -115,7 +115,7 @@ func (s *addressTestSuite) TestRandHexAccAddrConsistency() {
 	}
 
 	_, err := types.AccAddressFromHex("")
-	s.Require().Equal(types.ErrEmptyHexAddress, err)
+	s.Require().Contains(err.Error(), "empty address string is not allowed")
```

Comment:
We can move errors that we return into exported constants just like the original: types.ErrEmptyHexAddress

Reply:
Oh I can actually use that one, as this is the only usage, and change it to the message we have 👍 
Addressed.